lib/smart_answer_flows/state-pension-topup/questions/how_much_extra_per_week.govspeak.erb

```diff
@@ -6,6 +6,10 @@
   Enter a number between 1 and 25.
 <% end %>
 
-<% content_for :error_message do %>
-  Enter an amount in whole pounds between 1 and 25.
+<% content_for :error_not_whole_number do %>
+  Enter an amount which is a whole number.
```

Comment:
Following your suggestion, in the new version of this PR, I've changed both validation messages in the commit titled "Split weekly_amount validation into two validate blocks" (7cd20d938a893aab748148069da4ad6d046a28d5) as follows:

* "Enter an amount in whole pounds."
* "Enter an amount in pounds between 1 and 25."

I think this results in the smallest change from the existing behaviour/text.